src/invidious/yt_backend/extractors.cr

```diff
@@ -582,6 +588,135 @@ private module Parsers
     end
   end
 
+  # Parses an InnerTube lockupViewModel into a SearchPlaylist.
+  # Returns nil when the given object is not a lockupViewModel.
+  #
+  # This structure is present since November 2024 on the "podcasts" and
```

Comment:
Just a small nitpick about the dates, the podcasts tab has been like that since at least February 2024 as that is when I created the YouTube.js PR to handle it.

Reply:
Ah lol. It seems that nobody uses them, as I couldn't find an issue related to that!